build.gradle

```diff
@@ -353,15 +354,15 @@ dependencies {
         exclude group: "com.sun.xml.bind", module: "jaxb-osgi"
     }
 
-    testCompile(group: 'org.yaml', name: 'snakeyaml', version: '1.25') {
+    testCompile(group: 'org.yaml', name: 'snakeyaml', version: '1.26') {
```

Comment:
i think we can remove testCompile(group: 'org.yaml' as it is  compile group:"org.yaml", name: "snakeyaml", version:"1.26" and all will get compile.same for me also in my pr :)

Reply:
Done